Makefile

```
@@ -124,9 +124,16 @@ ifeq ($(NIM_PARAMS),)
 # The `git reset ...` will try to fix a `make update` that was interrupted
 # with Ctrl+C after deleting the working copy and before getting a chance to
 # restore it in $(BUILD_SYSTEM_DIR).
+ifeq ($(shell uname), Darwin)
+	LFS_MISSING_MSG := "Git LFS not installed. Run 'brew install git-lfs' to set up."
+else
+	LFS_MISSING_MSG := "Git LFS not installed."
+endif
```

You could just do the check here:
```suggestion
ifeq (, $(shell which git-lfs))
ifeq ($(shell uname), Darwin)
	$(error Git LFS not installed. Run 'brew install git-lfs' to set up.)
else
	$(error Git LFS not installed.)
endif
endif
```

---

Thanks. Had to remove the indentation to avoid `commands commence before first target` error